html/includes/graphs/application/freeradius_acct.inc.php

```diff
@@ -0,0 +1,29 @@
+<?php
+require 'includes/graphs/common.inc.php';
+$i            = 0;
+$scale_min    = 0;
+$nototal      = 1;
+$unit_text    = 'Per Sec.';
+$rrd_filename = rrd_name($device['hostname'], array('app', 'freeradius-acct', $app['app_id']));
+$array        = array(
+    'requests',
+    'responses',
+    'duplicate_requests',
+    'malformed_requests',
+    'invalid_requests',
+    'dropped_requests',
+    'unknown_types'
+);
+$colours      = 'mixed';
+$rrd_list     = array();
+if (rrdtool_check_rrd_exists($rrd_filename)) {
+    foreach ($array as $ds) {
+        $rrd_list[$i]['filename'] = $rrd_filename;
+        $rrd_list[$i]['descr']    = strtoupper($ds);
```

Comment:
Instead of strtoupper($ds) you could have pre-written descriptions:
Add descriptions to the array above (also a little more descriptive name than $array wouldn't be a bad idea)
```
'duplicate_requests' => 'Duplicate Requests,
```

Use them in the foreach
```
foreach ($array as $ds => $descr) {
        $rrd_list[$i]['filename'] = $rrd_filename;
        $rrd_list[$i]['descr'] = $descr;
```

Reply:
Sorry for all the mini-commits hitting the CI.  Internal dev env is so bad I've just been editing on the Github webUI.  :grimacing:

Anyway, I think I've got all recommendations included now.